Components/main_avionics.hpp

```diff
@@ -57,14 +60,13 @@ namespace SystemHandles {
 	constexpr UART_HandleTypeDef* UART_SOB = &huart1;
 	constexpr UART_HandleTypeDef* UART_Umbilical = &huart2;
 	constexpr UART_HandleTypeDef* UART_Radio = &huart3;
-	constexpr UART_HandleTypeDef* UART_Debug = &huart2;
+	constexpr UART_HandleTypeDef* UART_Debug = &huart3;
```

Comment:
Change it back to 2 (the umbilical) for now, eventually we'll have to get rid of SOAR_PRINT entirely from this board